recipe/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "azureml_mlflow" %}
+{% set version = "1.55.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/py3/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-py3-none-any.whl
+  sha256: 048cd60c483d778b1f52fc219a32c925706dceac871d829946099609959c0eef
+
+build:
+  number: 0
+  # skipping because of missing mlflow-skinny for s390x
+  skip: True  # [s390x]
+
+requirements:
+  host:
+    # patch tools are in host section since they are necessary only during execution of build script
+    - patch     # [not win]
+    - m2-patch  # [win]
+    - python
+    - pip
+    - wheel
+  run:
+    - python
+    - six
+    - cryptography
+    - mlflow-skinny
+    - azure-identity
+    - azure-common
+    - azure-core >=1.8.0,<2.0.0,!=1.22.0
+    - azure-mgmt-core >=1.2.0,<2.0.0
+    - azure-storage-blob >=12.5.0,<=12.19.0
+    - python-dateutil >=2.7.3,<3.0.0
+    - msrest >=0.6.18
+
+test:
+  requires:
+    - pip
+  commands:
+    - pip check
+  imports:
+    - azureml
+    - azureml.mlflow
+    - azure.core
+    - azure.mgmt.core
+    - azure.storage.blob
+    - mlflow
```

Comment:
```suggestion
```
The only thing we should test for is the package itself.

Reply:
since all these packages are dependencies for this one and this one is the "real" deliverable and none of those submodules are tested in their respective recipes, I thought it wouldn't hurt to test them here.

It wouldn't be the first time that some components ends up causing issues because of some missing dependency or because the dependency wasn't properly tested.